### DISCUSSION TITLE
Please add Microdox v2 ZMK support

### DISCUSSION THREAD

**Discussion by yooneo** [manna-harbour/miryoku#217] [Q&A] [answered]
Here is a ZMK shield that currently works for Microdox v2. Please consider adding it. https://github.com/ridingintraffic/zmk-config/tree/main/config/boards/shields/microdox

**Comment by manna-harbour** [comment, marked as answer]
Added in https://github.com/manna-harbour/miryoku_zmk/commit/3124961ba259b702eb694a2e85e92fd9f1a8659a. Here's a sample build run: https://github.com/manna-harbour/miryoku_zmk/actions/runs/5163363684.
